src/DotVVM.Framework/Controls/HtmlGenericControl.cs

```diff
@@ -194,6 +200,31 @@ private void AddCssClassesToRender(IHtmlWriter writer)
             if (cssClassBindingGroup != null) writer.AddKnockoutDataBind("css", cssClassBindingGroup);
         }
 
+        private void AddStylePropertiesToRender(IHtmlWriter writer)
+        {
+            KnockoutBindingGroup stylePropertiesBindingGroup = null;
+            foreach (var styleProperty in StyleProperties.Properties)
+            {
+                if (stylePropertiesBindingGroup == null) stylePropertiesBindingGroup = new KnockoutBindingGroup();
+                stylePropertiesBindingGroup.Add(styleProperty.GroupMemberName, this, styleProperty);
+
+                try
+                {
+                    var value = GetValue(styleProperty)?.ToString();
+                    if (!string.IsNullOrEmpty(value))
+                    {
+                        writer.AddStyleAttribute(styleProperty.GroupMemberName, value.ToString());
```

Comment:
You don't need to call `ToString` here.

Reply:
You're right, I forgot that after the last changes